openhub/portfolio_projects.py

```diff
@@ -0,0 +1,80 @@
+import json
+import requests
+import logging
+import xmltodict
+
+from community.git import get_owner
+from openhub.oh_token import OH_TOKEN
+from openhub.models import PortfolioProject, PortfolioProjectActivity
+
+
+def get_portfolio_projects_data():
+    n = 100
+    data1 = []
+    for i in range(1, n):  # Ignore PyFlakesBear
+        import_url = 'https://www.openhub.net/orgs/' + get_owner() \
+            + '/projects.xml?api_key=' + OH_TOKEN + '&page=' + str(i)
+        resp = requests.get(import_url)
+        try:
+            jsonString = json.dumps(xmltodict.parse(resp.content),
+                                    indent=4)
+            json_object = json.loads(jsonString)
+            data = json_object['response']['result'
+                                           ]['portfolio_projects']['project']
+        except Exception as ex:
+            logging.debug(ex)
+            break
+        data1 = data1 + data
+    return data1
+
+
+def import_data(project):
+    name = project.get('name', None)
+    activity = project.get('activity', None)
+    i_use_this = project.get('i_use_this', None)
+    community_rating = project.get('community_rating', None)
+    org = get_owner()
+    commits = project.get(
+        'twelve_mo_activity_and_year_on_year_change', None).get('commits', None)
```

Comment:
why does this `get` default to `None`, and then does another `get` on the result?
If it is `None`, the second get will fail.

Also you repeat `project.get('twelve_mo_activity_and_year_on_year_change', None)` lots of times.  Get it once, put it in a variable, and then use the variable.